arangosh/Restore/RestoreFeature.cpp

```diff
@@ -1610,9 +1616,22 @@ void RestoreFeature::start() {
 
   std::unique_ptr<SimpleHttpClient> httpClient;
 
+  auto const connectRetry = [&](size_t numRetries) -> Result {
+    for (size_t i = 0; i < numRetries; i++) {
```

Comment:
would it make sense to add an increasing sleep here? Hammering against the same error condition instantly will probably not improve the situation that much?